conf/logback-test.xml

```diff
@@ -50,12 +50,14 @@
 
 
     <logger name="application" level="INFO"/>
+    <logger name="uk.gov.hmrc.emcstfefrontend" level="DEBUG"/>
 
     <logger name="connector" level="ERROR">
         <appender-ref ref="STDOUT"/>
     </logger>
 
-    <root level="INFO">
+    <root level="ERROR">
         <appender-ref ref="FILE"/>
+        <appender-ref ref="STDOUT"/>
```

Comment:
Personal preference, but I normally leave STDOUT out of logback-test as it clutters up test output, and only put it in if I need to debug things like ITs. Happy to have a chat about this.

This is also why I don't mind keeping `<root level="INFO">` - unless explicitly changed to debug what's going on, it doesn't clutter up tests so it isn't an issue.

Reply:
Yeah, I can get that. That's why I always change the output format to take up less clutter and make it more meaningful. So it's a cleaner output than the out of the box one:
![image](https://user-images.githubusercontent.com/16954819/215087839-0a7ff35a-5c89-4ef1-ae9f-c2f01bbe4102.png)

I can remove it if you want, but I'll be constantly adding it back in and forgetting to remove it 😂

Reply:
I guess I don't like logs like that say things like **[WARN]** when the tests pass. Happy to leave in for now though, remind me to merge this on Monday 🙂